### PR TITLE
Add messages for invalid spring boot application

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

File: dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops [hunk 1]
```diff
@@ -49,4 +49,12 @@ error.missing.servlet.useraction=A Servlet feature needs to be configured in the
...
-error.multiple.applications.not.allowed.useraction=Configure only one Spring Boot application per server configuration.
\ No newline at end of file
+error.multiple.applications.not.allowed.useraction=Configure only one Spring Boot application per server configuration.
+
+error.no.manifest.found=CWWKC0256E: No META-INF/MANIFEST.MF file found for the Spring Boot application {0}.
+error.no.manifest.found.explanation=Spring Boot applications must contain a META-INF/MANIFEST.MF file.
+error.no.manifest.found.useraction=Configure a valid Spring Boot application with the server.
+
+error.no.spring.class.found=CWWKC0257E: The Spring Boot application META-INF/MANIFEST.MF file must contain a Start-Class header.
+error.no.spring.class.found.explanation=Spring Boot applications must a main class with the Start-Class manifest header.
```
Comment (dazavala) [inline dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops]: Explanation is missing a term, probably "contain".

Comment (tjwatson) [reply dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops]: I fixed up the message.

**Comment by tjwatson** [comment]
#build

**Pull request merged**